samples/tfm_integration/tfm_ipc/README.rst

```diff
@@ -103,6 +103,35 @@ Or, post build:
 
       $ ninja run
 
+On ST Nucleo L552ZE Q:
+======================
+
+Build Zephyr with a non-secure configuration:
+
+   .. code-block:: bash
+
+      $ west build -b nucleo_l552ze_q_non_secure samples/tfm_integration/tfm_ipc/
+
+Three scripts are generated in the ``build/tfm/install`` folder:
+
+  - ``postbuild.sh``: Updates regression.sh and TFM_UPDATE.sh scripts according to generated flash_layout.h.
+  - ``regression.sh``: Sets platform option bytes config and erase platform.
+  - ``TFM_UPDATE.sh``: Writes bl2, secure, and non secure image in target.
+
+Run them in the following order to flash the board:
+
+   .. code-block:: bash
+
+      $ ./build/tfm/install/postbuild.sh
+      $ ./build/tfm/install/regression.sh
+      $ ./build/tfm/install/TFM_UPDATE.sh
```

Comment:
We need to pass zephyr.bin path to this script so it is able to flash the correct _ns binary and not the default tfm_ns one.

Reply:
It is too complicated to automate the process of calling these scripts during the `extra_post_build_commands` step, similar to how we sign the images with a script post-build?

Reply:
Calling these scripts can be done during  `extra_post_build_commands`. However, the scripts `regression.sh` and `TFM_update.sh` are used to flash the board, not building the binaries, so I think that it is better to keep them as a separate step.

Reply:
The latest push modifies the scripts, to flash zephyr ns image instead of the tfm ns image, without passing the path or editing the script.

Reply:
If `postbuild.sh` can be called automatically without modification, it should be added to the board's `CMakeLists.txt` file in the `extra_post_build_commands` section, and the documentation updated to list only the two scripts to run manually.

A small description of why they are necessary and what they do may also be useful since the names themselves aren't very clear or obvious (`regression.sh`?).